routes/spec.js

```diff
@@ -618,13 +619,16 @@ Currently the API has a rate limit of **60 requests/minute** and **50,000 reques
           },
         },
         route: () => '/guilds/:player',
-        func: (req, res) => {
-          getGuildFromPlayer(req.params.player, req.query.populatePlayers, (err, guild) => {
-            if (err) {
-              return res.status(404).json({ error: err });
+        func: async (req, res, cb) => {
```

Comment:
Should be `request`, `response`, and `callback` to keep it consistent with the other changes

Reply:
Fixed!